### PR TITLE
REG-881: Find all Local Units for enterprise and period

### PR DESCRIPTION
**Pull request by nigelhp**

### CODE REVIEW

File: app/utils/EitherSupport.scala [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package utils
+
+object EitherSupport {
```
Comment (ONS-Tom) [inline app/utils/EitherSupport.scala]: It would be great to have a short comment here explaining things

Comment (nigelhp) [reply app/utils/EitherSupport.scala]: scaladoc API comment, along with implementation specific comment added.

File: app/repository/hbase/localunit/HBaseRestLocalUnitRepository.scala [hunk 2]
```diff
@@ -25,16 +26,32 @@ class HBaseRestLocalUnitRepository @Inject() (
 
   override def retrieveLocalUnit(ern: Ern, period: Period, lurn: Lurn): Future[Either[ErrorMessage, Option[LocalUnit]]] = {
     logger.info(s"Retrieving Local Unit with [$ern] [$lurn] for [$period].")
-    restRepository.findRow(config.tableName, LocalUnitRowKey(ern, period, lurn), DefaultColumnGroup).map(fromErrorOrRow)
+    restRepository.findRow(config.tableName, LocalUnitQuery.byRowKey(ern, period, lurn), DefaultColumnGroup).map(fromErrorOrRow)
```
Comment (ameenhaq) [inline app/repository/hbase/localunit/HBaseRestLocalUnitRepository.scala]: Still not too convinced with DefaultColumnGroup. The DB conventional name for column group name is ColumnFamily - maybe rename its to 'DefaultColumnFamily' to keep the naming consistent from DB to client

File: app/repository/hbase/HBase.scala [hunk 3]
```diff
@@ -2,6 +2,7 @@ package repository.hbase
 
 object HBase {
   val RowKeyDelimiter = "~"
+  val Wildcard = "*"
   val DefaultColumnGroup = "d"
```
Comment (ameenhaq) [inline app/repository/hbase/HBase.scala]: Still not too convinced with DefaultColumnGroup. The DB conventional name for column group name is ColumnFamily - maybe rename its to 'DefaultColumnFamily' to keep the naming consistent from DB to client

**Pull request merged**